# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: d0bb4f6201a0
# Full Hash: d0bb4f6201a04831e67ba1f1f4591d96586fd8f0
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p9 - Move nsPrintData::mPrintObject to nsPrintJob. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146980
# ==============================================================================

diff -r 853cf247e870 -r d0bb4f6201a0 layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Sun May 22 09:37:58 2022 +0000
+++ b/layout/printing/nsPrintJob.h	Sun May 22 09:37:58 2022 +0000
@@ -255,6 +255,9 @@
   // Only set if this nsPrintJob was created for a real print.
   RefPtr<RemotePrintJobChild> mRemotePrintJob;
 
+  // The root print object.
+  mozilla::UniquePtr<nsPrintObject> mPrintObject;
+
   // If there is a focused iframe, mSelectionRoot is set to its nsPrintObject.
   // Otherwise, if there is a selection, it is set to the root nsPrintObject.
   // Otherwise, it is unset.
